apps/daimo-web/src/components/Providers.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
+
+import {
+  RainbowKitProvider,
+  getDefaultWallets,
+  connectorsForWallets,
+} from "@rainbow-me/rainbowkit";
+import * as React from "react";
+import { configureChains, createConfig, WagmiConfig } from "wagmi";
+import { baseGoerli } from "wagmi/chains";
+import { publicProvider } from "wagmi/providers/public";
+
+const { chains, publicClient, webSocketPublicClient } = configureChains(
+  [baseGoerli],
+  [publicProvider()]
+);
+
+const walletConnectProjectId = "06d134eadbbacc1f9dd4575541dda90c";
```

Comment:
i don't fully understand this part

RainbowKit handles metamask etc out of the box

is this so that it additionally handles WC?

Reply:
yep, injected providers work out of the box, wallet connect requires this (connecting phone rainbow -> browser website etc.)